### PR TITLE
Working page

### PR DESCRIPTION
**Pull request by eypsrcnuygr**
There is an issue in the package.lock.json that we did not understand!
This page is a heat map of the Smashing Magazine.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    <a class="nav-element">Membership</a>
+    <a class="nav-element">Jobs</a>
+    <a class="nav-element">Newsletter</a>
+    <a class="nav-element">Podcast</a>
+    <a class="nav-element">Write for us</a>
+    <a class="nav-element">More</a>
+    <form class="nav-element"><input/></form>
+  </nav>
+  <main>
+    <div class="top-section">
```
Comment (elmejdki) [inline index.html:26]: this section should only contain one article, you make it have three looks at the original page:
#### clone:
image

#### original:
image

I think you have a bigger screen size always make sure that you are zoomed in cause most users have a lower screen size computer.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+    </div>
+    
+  </main>
+  <footer>
+    <h4>BROWSE ALL SMASHING MAGAZINE TOPICS</h4>
+    <p>Accessibility . Android . Animation . Apps . CSS Design . Design Patterns <br>
+      Design . Systems . E-Commerce . Freebies . Graphics . HTMLIllustrator . Inspiration <br>
+      iOS . JavaScript . MobilePattern . Libraries . Performance . Photoshop . Plugins <br>
+      React . Responsive Web Design . Service Workers  . Sketch . Typography . UI <br>
+      Usability . User Experience . Wallpapers . Web Design . WordPress . Workflow</p> <br>
```
Comment (elmejdki) [inline index.html:319]: each item inside the paragraph should be a link please wrap them inside a link tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+      <p><small>With a commitment to quality content for the design community. <br>
+
+        Founded by Vitaly Friedman and Sven Lennartz. 2006–2020. <br>
+        
+        Smashing is proudly running on Netlify. <br>
+        
+        Fonts by Latinotype.</small></p>
+        <p><small>✎ Write for usContact usAbout us (Impressum)Privacy policy</small></p>
```
Comment (elmejdki) [inline index.html:327]: these also should be links write to us, contact us, ...

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: styles/styles.css [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+  grid-template-columns: 1fr 1fr 2fr;
+  justify-items: center;
+  padding-left: 5%;
+  padding-right: 20%;
+  grid-gap: 15px;
+}
+
+.fourth-section > :nth-child(3) {
+  background-color: #d33a2c;
+}
```
Comment (andresporras3423) [inline styles/styles.css:109]: This selector has the same style definition of selector on line 94. Group selectors with the same style definition to minimize your code.

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: styles/styles.css [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+  width: 100px;
+  height: 100px;
+  background-color: #a0a0a0;
+}
+
+.seventh-section article {
+  display: flex;
+}
+
+#week {
```
Comment (andresporras3423) [inline styles/styles.css:134]: Don't overuse ID selectors

**Comment by eypsrcnuygr** [comment]
Id's are changed into class and semantic selectors.
Colors changed into grayscale.
Linter checked.
Validator checked.

**Pull request merged**